packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterForm.tsx

```diff
@@ -263,11 +383,17 @@ function GridFilterForm(props: GridFilterFormProps) {
             </option>
           ))}
         </rootProps.components.BaseSelect>
-      </rootProps.components.BaseFormControl>
-      <rootProps.components.BaseFormControl
+      </FilterFormOperatorInput>
+      <FilterFormValueInput
         variant="standard"
-        sx={{ width: 190 }}
-        {...rootProps.componentsProps?.baseFormControl}
+        as={rootProps.components.BaseFormControl}
+        {...baseFormControlProps}
+        {...valueInputProps}
+        className={clsx(
+          classes.valueInput,
```

Comment:
Maybe this has been already answered but what about when the `FilterFormValueInput` is the select or the multi-select, will it always has that `classes.valueInput` class or should we change the class based on the component?

Reply:
Not already answered. Modifying the class according to the input seems tricky. 
We should add a property `currentOperator.valueFormControlClass`

In my opinion, it is overkilled compared to what most of the dev will probably do with the customization:
- removing some option
- modifying input size

About the size, `classes.valueInput` is useful to remove default one. To have customer size depending on the input component, I see two cases:
- it is their own input, so they can do whatever they want
- it is one of the provided input. They can distinguish single and multiple values with CSS controller, depending on `MuiFormControl-root` and `MuiInput-root`

Reply:
I'm not sure if I'm on the same page here but `classes.valueInput` is applied to the `FormControl` that wraps the custom filter input component. In this context, `FilterFormValueInput` will always be a `FormControl`, unless the user changes it through the slots.

Reply:
Yes, we are on the same page

For now, the `FormControl` is responsible for the width. So to customize the `classes.valueInput` must be used to add `width:'unset'`